blocks/library/latest-posts/index.js

```diff
@@ -33,6 +33,8 @@ registerBlockType( 'core/latest-posts', {
 
 	category: 'widgets',
 
+	keywords: [ __( 'recent-posts' ) ],
```

Comment:
I thought spaces wouldn't work, otherwise `recent posts` would signal you can do `tagA tagB tagC ... tagN` and circumvent the 3 keywords max :)